src/styles/partials/components/_summary-list.scss

```diff
@@ -0,0 +1,67 @@
+.dg_summary-list {
+  color: $gray;
+  background-size: cover;
+  background-color: $gray-lightest;
+}
+  .dg_summary-list-items {
+    display: flex;
+    align-items: flex-start;
+    flex-wrap: wrap;
+  p {
+    margin-bottom: 0px;
+  }
+  .hide {
+    visibility: collapse;
+  }
+  a {
+    text-decoration: none;
+    background-color: transparent;
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+}
+.dg_summary-list-items > * {
+  flex: 1 100%;
+  padding: $padding-small;
+}
+
+.heading {
+  margin-top:$padding-small;
+}
+.icon-1 {
+  font-size: $h1-font-size;
+  color:$gray ;
+}
+.push{
+   visibility: collapse;
```

Comment:
I'm not familiar with what this is doing, why did you choose to use this?

Reply:
The visibility property in CSS has two different functions. It hides rows and columns of a table, and it also hides an element *without changing the layout*.